product_docs/docs/pgd/5/rel_notes/pgd_5.4.0_rel_notes.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "EDB Postgres Distributed 5.4.0 release notes"
+navTitle: "Version 5.4.0"
+---
+
+Released: 05 Mar 2024
+
+EDB Postgres Distributed version 5.4.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.4. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+## Highlights of EDB Postgres Distributed 5.4.0
+
+* Highlights needed
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
+
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres Advanced Server or EDB Postgres Extended 
+upgrades with an upgrade of EDB Postgres Distributed.
+!!!
+
+## Features
+
+| Component | Version | Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| BDR       | 5.4.0   | Automatically detect and synchronize all available nodes to the furthest ahead node for transactions originating from failed or disconnected node. ||
+| BDR       | 5.4.0   | Automatic resolution of pending group commit transactions when the originating node fails or disconnects, ensuring uninterrupted transaction processing within the cluster. ||
+| BDR       | 5.4.0   | Added ability to set `apply_delay` group option on sub-groups, enabling adding of delayed subscriber-only nodes | | /* 25-apply-delay */
+
+
+## Enhancements
+
+| Component | Version  |Description | Addresses |
+|-----------|---------|-------------|-----------|
+
+## Bug fixes
+
+| Component | Version |Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| BDR       | 5.4.0   | Fixed memory leaks when running a query on some or all nodes. |  | /* 10-Memory-Leak-Run-On-All-Nodes */
+| BDR       | 5.4.0   | Resolved an issue of high CPU usage for concensus processes | RT97649 | /* 20-High-CPU-usage-consensus */
```

Comment:
There's still a typo here: "concensus" should be "consensus"

Reply:
Fixed